NOTES.txt

```diff
@@ -2,4 +2,6 @@ This is a notes document.
 
 There are many notes contained in this document.
 
-Here is an additional note that is being added.
\ No newline at end of file
+Here is an additional note that is being added.
+
+Adding another note to the file.
```

Comment:
Here is a cool pic.

![image](https://cloud.githubusercontent.com/assets/5218961/23372794/5b2d6426-fcda-11e6-948a-fc291755e241.png)